File: Plataformas/Codeforces/Round 1016 (Div. 3)/B. Expensive Number.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int t;
    cin >> t;

    while(t--) {
        string n;
        cin >> n;
        int index_sobra = 0;
        int ans = 0;
        for (int i = n.size() - 1; i >= 0; i--) {
            if (n.at(i) != '0') { 
                index_sobra = i; 
                break;
            }
        }

        for (int i = index_sobra - 1; i >= 0; i--) 
            if (n.at(i) != '0') ans++;

        ans += n.size() - 1 - index_sobra;

        cout << ans << "\n";
    }
}

File: Plataformas/Codeforces/Round 1016 (Div. 3)/B. Expensive Number.py
t = int(input())

for _ in range(t) :
    n = input()
    ans = 0
    index_sobra = 0

    for i in range(len(n) - 1, -1, -1) :
        if (n[i] != '0') :
            index_sobra = i
            break

    for i in range(index_sobra - 1, -1, -1) :
        if n[i] != '0' : ans += 1
    
    ans += len(n) - 1 - index_sobra
    print(ans)
    